compiler/ras/Debug.cpp

```diff
@@ -1967,6 +1967,17 @@ TR_Debug::getStaticName(TR::SymbolReference * symRef)
       if (sym->isConst())
          return "<constant>";
 
+      // Value Type default value instance slot address
+      if (sym->isStatic() && staticAddress)
+         {
+         char * name = (char *)_comp->trMemory()->allocateHeapMemory(TR::Compiler->debug.pointerPrintfMaxLenInChars()+5);
```

Comment:
Why do we have a `+5` here? Probably best to either use a const variable or a macro definition so that it's not just a magic number.

Reply:
That's a good question. I was following the example on how currently the `staticAddress` is printed. I can change it to a const definition. https://github.com/eclipse/omr/blob/addc9c5c961f8bddc8ce2aa9a87394450ea027bc/compiler/ras/Debug.cpp#L1997-L2004

Reply:
Addressed in 1cd0e50